index.html

```diff
@@ -175,5 +175,33 @@ <h2 class="tit-abi">Skills</h2>
      </section>  
    </footer>
 
+   <div id="form-contact1">
+      <p class="title" id="contact-tit1">Get started</p>
+      <p class="parrafo" id="descr-contact1">
+         footer pc - Lorem ipsum dolor sit amet, consectetur adipiscing elit,
+          sed do eiusmod tempor incididunt ut labore et dolore magna
+          aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+          ullamco laboris nisi ut aliquip ex ea commodo consequat. 
+      </p>
+      <div class="form-container">
+         <form name="contact-form" action="https://formspree.io/f/mzbwzdgd" 
+             method="post" id="contact-form1">
+            <div id="line1-cf">
+               <div id="name-group1">
+              <input class="dis-block"  id="name-form1" name="name" type="text" placeholder="Your full name" maxlength="30" required>
+               </div>
+               <div id="email-group1">
+                  <input class="dis-block"  id="email-form1" name="email" type="email" placeholder="Your email address" required>
+               </div>
+            </div>   
+            <div id="message-group">
+               <textarea class="dis-block"  id="message-form1" name="message" placeholder="Enteryour message here"
+                  maxlength="500" required></textarea>
+            </div>      
```

Comment:
- [ ] Kindly style your contact form as the given Figma template.

Yours:
![image](https://user-images.githubusercontent.com/92215878/187497310-f716e9db-551e-49bd-87c6-eceb59f1d005.png)


Figma:
![image](https://user-images.githubusercontent.com/92215878/187497226-b2c551c1-7477-42d4-a10a-d53205cdf3d9.png)